From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Cheng Zhao <zcbenz@gmail.com>
Date: Wed, 4 Mar 2020 14:23:28 -0700
Subject: fix: linux arm build

Fix Linux ARM build error caused by compiler update.

This means to be a quick fix, we should choose other solution once we
get an answer at:
https://bugs.chromium.org/p/chromium/issues/detail?id=1058107

diff --git a/third_party/marl/src/osfiber_asm_arm.S b/third_party/marl/src/osfiber_asm_arm.S
index d276ac806..b2be5face 100644
--- a/third_party/marl/src/osfiber_asm_arm.S
+++ b/third_party/marl/src/osfiber_asm_arm.S
@@ -22,6 +22,7 @@
 // x1: to
 .text
 .global marl_fiber_swap
+.type marl_fiber_swap, %function
 .align 4
 marl_fiber_swap:
 
